README.md

```diff
@@ -4,17 +4,26 @@ Project repository template to set up all public Phoenix projects at [Nimble](ht
 
 ## Installation
 
-NimblePhxGenTemplate is published on Hex. Add it to your list of dependencies in `mix.exs`:
+### Generate a fresh new Phoenix project by using [mix phx.new](https://hexdocs.pm/phoenix/Mix.Tasks.Phx.New.html)
```

Comment:
The word `fresh new` sounds redundant to me. How about `a new Phoenix project`

Reply:
Thanks @hanam1ni, I have updated it in https://github.com/nimblehq/elixir-templates/pull/73/commits/41f8dda6c05b9e6f4ce1be99fb760091dd43900f :)